docs/README.md

```diff
@@ -1,12 +1,23 @@
 # Doctrine ORM Documentation
 
 ## How to Generate:
-Using Ubuntu 14.04 LTS:
+
+The following will generate the documentation into the `build` directory of the checkout.
+
+### Using Ubuntu 14.04 LTS:
 
 1. Run ./bin/install-dependencies.sh
 2. Run ./bin/generate-docs.sh
 
-It will generate the documentation into the build directory of the checkout.
+### Using macOS:
+
+Sphinx Installation instructions can be found at the
```

Comment:
This is no longer needed with the new website, tbh /cc @jwage

Reply:
ah nice even better; just stumbled upon this when I wanted to verify #7185 and wanted to share my insights :) I realized that it gets rendered in the old-school style, but for veryfing the linking it was fair enough.

Reply:
This is updated here https://github.com/doctrine/doctrine2/pull/7177

Reply:
ok, I guess mine can be closes then. Thanks!